tests/test_study.py

```diff
@@ -392,10 +393,15 @@ def f(trial):
     with StorageSupplier(storage_mode) as storage:
         study = optuna.create_study(storage=storage)
         study.optimize(f, n_trials=3)
-        df = study.trials_dataframe()
+        print(study.trials[0].params_in_internal_repr)
```

Comment:
Is this line necessary?

Reply:
Sorry for my careless mistake. I'll remove it. Thank you.